app/views/search_page/search_page.erb

```diff
@@ -6,6 +6,11 @@
     locale: I18n.locale,
     defaultLocale: I18n.default_locale
   },
+  marketplace: {
+    marketplace_color1: CommonStylesHelper.marketplace_colors(@current_community)[:marketplace_color1],
```

Comment:
I know I'm behind the original naming, but `marketplace.marketplace_color1` feels a bit redundant